projects/core/src/util/get-last-value-sync.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { getLastValueSync } from '@spartacus/core';
+import { EMPTY, from, NEVER, of } from 'rxjs';
+
+describe('getLastValueSync', () => {
+  it('should return undefined if there is no emission', () => {
+    expect(getLastValueSync(EMPTY)).toBeUndefined();
+    expect(getLastValueSync(NEVER)).toBeUndefined();
+  });
+
+  it('should return value synchronously', () => {
+    expect(getLastValueSync(of('test'))).toEqual('test');
+  });
+
+  it('should return last value synchronously', () => {
+    expect(getLastValueSync(from(['a', 'b', 'c']))).toEqual('c');
+  });
+});
```

Comment:
Specs are kind of documentation. What about adding tests:
- `should return undefined when observable is not synchronous` - `of('test', asyncScheduler)`
- `should return undefined for hot observable` - `subject$.next('test'); getlastValueSync(subject$)`